apps/api/src/courses/__tests__/course.controller.e2e-spec.ts

```diff
@@ -843,27 +844,30 @@ describe("CourseController (e2e)", () => {
             .set("Cookie", cookies);
 
           expect(response.status).toBe(200);
-          expect(response.body).toEqual({
-            data: [
-              {
-                firstName: students[0].firstName,
-                lastName: students[0].lastName,
-                email: students[0].email,
-                id: students[0].id,
-                enrolledAt: studentCourse[0].createdAt,
-                groups: [],
-                isEnrolledByGroup: false,
-              },
-              {
-                firstName: students[1].firstName,
-                lastName: students[1].lastName,
-                email: students[1].email,
-                id: students[1].id,
-                enrolledAt: null,
-                groups: [],
-                isEnrolledByGroup: false,
-              },
-            ],
+          expect(response.body.data).toEqual([
+            {
+              firstName: students[0].firstName,
+              lastName: students[0].lastName,
+              email: students[0].email,
+              id: students[0].id,
+              enrolledAt: studentCourse[0].createdAt,
```

Comment:
```suggestion
              enrolledAt: studentCourse[0].enrolledAt,
```
Please change in all the tests using this